components/size_aware_image.jsx

```diff
@@ -178,18 +263,57 @@ export default class SizeAwareImage extends React.PureComponent {
 
             return (
                 <div
-                    onClick={this.handleImageClick}
-                    className={className}
-                    style={this.state.imageWidth > MIN_IMAGE_SIZE ? {
-                        width: this.state.imageWidth + 2, // 2px to account for the border
-                    } : {}}
+                    className={classNames('small-image-utility-buttons-wrapper')}
                 >
-                    {image}
+                    <div
+                        onClick={this.handleImageClick}
+                        className={className}
+                        style={this.state.imageWidth > MIN_IMAGE_SIZE ? {
+                            width: this.state.imageWidth + 2, // 2px to account for the border
+                        } : {}}
+                    >
+                        {image}
+                    </div>
+                    <span
+                        className={classNames('image-preview-utility-buttons-container', 'image-preview-utility-buttons-container--small-image')}
+                        style={this.state.imageWidth > MIN_IMAGE_SIZE ? {
+
+                            // for every pixel the image is wider than MIN, add that to left shift of buttons container
+                            // 20px is the width of collapse button.
+
+                            left: 26 + (this.state.imageWidth - MIN_IMAGE_SIZE),
```

Comment:
This might not be something we can improve easily, but is it not possible to position this using a `right` value instead of doing this math here? Constants like this always have me worried that something will change slightly in the CSS and subtly break this without anyone noticing

Reply:
![absolute_right](https://user-images.githubusercontent.com/22407757/152463415-38f40ef0-792c-424f-b885-ad28701315cd.PNG)

When I started out with right, this happened.

I'll try again later today, what you said definitely makes sense to me. Thanks for the review. I've done other changes.

Reply:
You were 'right' :D , when I explicitly defined position of the parent container to relative, `right` tag started working. Please review the latest commit.

Reply:
> This might not be something we can improve easily, but is it not possible to position this using a `right` value instead of doing this math here? Constants like this always have me worried that something will change slightly in the CSS and subtly break this without anyone noticing

 @hmhealey this alignment had broken the small image wrapper. I have used left alignment again in the latest commit.